src/mono/wasm/runtime/startup.ts

```diff
@@ -690,7 +690,7 @@ export async function mono_wasm_load_config(configFilePath: string): Promise<voi
         const configRaw = await runtimeHelpers.fetch(configFilePath);
         const config = await configRaw.json();
 
-        runtimeHelpers.config = config;
+        runtimeHelpers.config = config || {};
         config.environment_variables = config.environment_variables || {};
```

Comment:
if config wasn't an object, all of the statements after this are going to throw since we're doing `config.` instead of `runtimeHelpers.config.`